odl/deform/linearized.py

```diff
@@ -135,6 +149,19 @@ def __init__(self, template, domain=None):
             this option is useful mainly when using different interpolations
             in displacement and template.
             Default: ``template.space.real_space.tangent_bundle``
+        interp : str or sequence of str
+            Interpolation type that should be used to sample the template on
+            the deformed grid. A single value applies to all axes, and a
+            sequence gives the iterpolation scheme per axis.
+
+            Supported values: ``'nearest'``, ``'linear'``
+
+            .. note::
```

Comment:
Do you mean `..warning::` or a `stderr` warning? I'm not a big fan of the latter since they are annoying for the "I know what I'm doing" kind of folks. And here it only affects the (admittedly rather important) case of operator derivative.
I'll go with the `..warning::` for the docs.